<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingClass</class>
 <widget class="QMainWindow" name="SettingClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/DealBox.qrc">
    <normaloff>:/image/Setting.svg</normaloff>:/image/Setting.svg</iconset>
  </property>
  <widget class="QWidget" name="Widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="Layout" rowstretch="5,3,2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="2" column="0">
     <widget class="QGroupBox" name="About_Box">
      <property name="title">
       <string>关于DealBox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="About_Box_Layout">
       <item row="3" column="0">
        <widget class="QLabel" name="Statement">
         <property name="text">
          <string>All rights reserved (c) 2024 zhongzhenjie0729@outlook.com</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="About">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="markdown">
          <string/>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;h1 style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:xx-large; font-weight:700;&quot;&gt;&lt;br /&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="About_QT">
         <property name="text">
          <string>关于QT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="Update_ExplainBox">
      <property name="title">
       <string>更新</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="Update_ExplainBox_Layout">
       <item row="1" column="0">
        <widget class="QTextEdit" name="Update_Explain">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="Version_Layout">
         <item>
          <widget class="QLabel" name="Current_Version">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Check_Update">
           <property name="text">
            <string>检查更新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="Setting_Box">
      <property name="title">
       <string>设置</string>
      </property>
      <layout class="QGridLayout" name="Setting_Box_Layout">
       <item row="0" column="0">
        <widget class="QComboBox" name="Select_Theme">
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>主题</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resource/DealBox.qrc"/>
 </resources>
 <connections/>
</ui>
